yarn.lock

```diff
@@ -294,7 +294,6 @@
 "@kubernetes/client-node@0.11.2":
   version "0.11.2"
   resolved "https://registry.yarnpkg.com/@kubernetes/client-node/-/client-node-0.11.2.tgz#5dbc1d66d3c954af0f3bd294dece884737f5b761"
-  integrity sha512-Uhwd2y2qCvugICnHRC5h2MT5vw0a1dJPVVltVwmkeMuyGTPBccsTtpTcSfSLitwOrh4yr+9wG5bRcMdgeRjYPw==
```

Comment:
I don't know why but we have a lot of removed lines like this

Reply:
Strange, I just use yarn as usual.... Will revert these changes

Reply:
From what I see, yarn deletes them automatically.

Reply:
> integrity sha512-Uhwd2y2qCvugICnHRC5h2MT5vw0a1dJPVVltVwmkeMuyGTPBccsTtpTcSfSLitwOrh4yr+9wG5bRcMdgeRjYPw==

This thing generation depends on yarn version